### DISCUSSION TITLE
Replace CSS stylesheet of the webapp

### DISCUSSION THREAD

**Discussion by alvitali** [lldap/lldap#855] [Q&A] [answered]
Is there a way to replace the stylesheet for the web UI of lldap with a custom one? I love lldap, but I don't love the UI. Would be great if I could write custom CSS for it.

Could I just map a folder in docker with the correct path, or would I need to build the docker image myself?

**Comment by nitnelave** [comment, marked as answer]
The looks of the frontend are defined by the contents of the /data/static folder. If you want to change them, you'll have to replace the index.html (since it pulls bootstrap) and add your own CSS in there.

The HTML in general is styled with bootstrap classes, you can reuse that for styling.

If you make something that looks nice, feel free to post it and I might either link to it or (with your permission) make it the LLDAP main theme!

**Reply by alvitali** [reply]
Thanks a lot! I'll look into it - will let you know! :)
